### ISSUE TITLE
voc数据集训练totallossnan lossnan问题？

### ISSUE THREAD

**Issue by ChenMaolong** [bubbliiiing/yolo3-pytorch#155]
大佬，我用的model_path      = 'model_data/yolo_weights.pth'跑的voc单类别数据集，类别也改了，
Start Validation
Epoch 73/600: 100%|██████████| 14/14 [00:08<00:00,  1.66it/s, val_loss=nan]
Finish Validation

Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.
Epoch:73/600
Total Loss: nan || Val Loss: nan 
Start Train
Epoch 74/600: 100%|██████████| 118/118 [00:28<00:00,  4.10it/s, loss=nan, lr=0.000965]
Finish Train
Start Validation
Epoch 74/600: 100%|██████████| 14/14 [00:08<00:00,  1.64it/s, val_loss=nan]
Finish Validation

Intel MKL ERROR: Parameter 6 was incorrect on entry to DGELSD.
出现这种问题？是因为预训练权重的原因吗但是你github只有coco的权重有voc的吗？

**Comment by bubbliiiing** [comment]
能直接用

**Comment by ChenMaolong** [comment]
> 能直接用

好的，谢谢大佬
